### ISSUE TITLE
Updates for proper handling of LatLong columns

### ISSUE THREAD

**Issue by thehomebrewnerd** [alteryx/woodwork#568]
In the previous Woodwork implementation using DataTables and DataColumns, any columns that were designates as LatLong columns would be reformatted and properly stored in a standard format. We need to implement this functionality in the new accessor approach as well. 

Note that reformatting of a standalone series cannot be done in place, and may require the use of the helper function created in #554.

**Comment by tyler3991** [comment]
@thehomebrewnerd , putting this to top of dev backlog. Let me know if it should go somewhere else.

**Comment by thehomebrewnerd** [comment]
@tyler3991 This is something we will need for feature parity with Woodwork prior to the accessor implementation, but I feel like this is lower priority that many of the other issues, since it only impacts one very specific data type, which isn't as common as most others.

**Comment by tyler3991** [comment]
Ok, got it. Thanks @thehomebrewnerd.

**Comment by thehomebrewnerd** [comment]
This code can be adapted for this conversion:
```
    if _get_ltype_class(self.logical_type) == LatLong:
        # Reformat LatLong columns to be a length two tuple (or list for Koalas) of floats
        if dd and isinstance(self._series, dd.Series):
            name = self._series.name
            meta = (self._series, tuple([float, float]))
            self._series = self._series.apply(_reformat_to_latlong, meta=meta)
            self._series.name = name
        elif ks and isinstance(self._series, ks.Series):
            formatted_series = self._series.to_pandas().apply(_reformat_to_latlong, use_list=True)
            self._series = ks.from_pandas(formatted_series)
        else:
            self._series = self._series.apply(_reformat_to_latlong)
```

**Comment by thehomebrewnerd** [comment]
Closed by #598
